pyproject.toml

```diff
@@ -121,6 +121,8 @@ netaddr = "~0.8.0"
 # Note: netutils is limited in scope, dependencies, and observes semver, as such
 #       we permit a looser (^) version constraint here.
 netutils = "^1.4.1"
+# Handling of version numbers
+packaging = "~23.0"
```

Comment:
This will upgrade `packaging` than what's currently installed. Desired?

Reply:
It's what we have in `next` as of #3563... as long as tests pass I think it should be fine. I don't see anything concerning in https://packaging.pypa.io/en/stable/changelog.html.

Reply:
I blame this issue for being the one for me to be overly paranoid about lockfile diffs 🤣.